src/server/index.tsx

```diff
@@ -201,6 +203,9 @@ server.get("/*", async (req, res) => {
            <!-- A remote debugging utility for mobile -->
            ${erudaStr}
 
+           <!-- Custom injected script -->
+           ${customScript}
```

Comment:
Should be called like `customHtmlHeader` if its like this. For `customScript`, i would expect something like `<script src="${customScript}"></script>`.

Reply:
Agree... how about `LEMMY_UI_CUSTOM_HTML`? 

I'll make an issue to document the ENV vars.

Reply:
I would use `LEMMY_UI_CUSTOM_HTML_HEADER` to make it a bit clearer.